src/auth.ts

```
@@ -9,6 +9,7 @@ import { checkIfUserExists } from "~/server/_utils/auth/auth";
 import { generateUniqueInviteCode } from "~/server/_utils/invite";
 import { USER_ROLES, WAITLIST_APPROVED_CODE } from "~/server/_utils/roles";
 import { logger } from "~/_utils";
+import { FeatureFlags, getFeatureFlagVariant } from "~/app/_lib/posthog";
```

ESLint

[error] `~/app/_lib/posthog` import should occur before import of `~/env`

(import/order)